MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -14148,7 +14154,14 @@ Folder *ApplicationWindow::appendProject(const QString &fn,
 
   // Open folders
   ProjectSerialiser serialiser(this);
-  serialiser.load(lines, fileVersion);
+
+  try {
+    serialiser.load(lines, fileVersion);
+  } catch (std::runtime_error &e) {
+    g_log.error(e.what());
+    // We failed to load - bail out
+    return nullptr;
```

Comment:
I don't understand why this method needs to return anything - it doesn't ever seem to get assigned to, and it only returns `nullptr`. Am I missing something, or could it be changed to `void`?

Reply:
I'm matching the existing returns for this method. So I'm not sure why we are returning handles, only the fact that we need to (unless we want to change multiple callers in ApplicationWindow)